core/tests/test_utils.py

```diff
@@ -84,19 +87,6 @@
 LOG_LINE_PREFIX = b'LOG_INFO_TEST: '
 
 
-def empty_environ():
```

Comment:
I have removed this because it is hacky. We should not modify the environ values directly, they are an implementation detail of testbed.

testbed already does the job of setting up the environ values, and it also cleans them up when `.deactivate()` is called.